### ISSUE TITLE
Empty movie list with Radarr

### ISSUE THREAD

**Issue by vysmaty** [morpheus65535/bazarr#1683]
**Describe the bug**
The movie list is empty. The connection to Radarr is working. Movie titles are visible in the debugg log.
| Sonarr is working. Series are listed.

**To Reproduce**
1. Task: Update Movie list from Radarr
2. Click on 'Movies' -> "No Movies Found"
3. Go to log

**Expected behavior**
Show list of movies.

**Software (please complete the following information):**
 - Bazarr: v**1.0.3-beta.11**
 - Radarr version 3.2.2.5080
 - Sonarr version [e.g. v 2.0.0.0001]
 - OS: Docker / linuxserver.io

**Additional context**

**Bazarr**: v1.0.3-beta.11
**Radarr**: 3.2.2.5080
**Sonarr**: 3.0.6.1342

**Docker: Console**
```
2022-01-15 20:48:35,631 - root                             (7f5c1556db38) :  INFO (signalr_client:40) - BAZARR trying to connect to Sonarr SignalR feed...

2022-01-15 20:48:35,632 - root                             (7f5c1546ab38) :  INFO (signalr_client:162) - BAZARR trying to connect to Radarr SignalR feed...

2022-01-15 20:48:35,632 - root                             (7f5c23146b48) :  INFO (server:37) - BAZARR is started and waiting for request on http://0.0.0.0:6767

2022-01-15 20:48:35,828 - root                             (7f5c15367b38) :  INFO (signalr_client:186) - BAZARR SignalR client for Radarr is connected and waiting for events.

2022-01-15 20:48:36,242 - root                             (7f5c1556db38) :  INFO (signalr_client:54) - BAZARR SignalR client for Sonarr is connected and waiting for events.

2022-01-15 20:48:54,730 - app                              (7f5c1a0cbb38) :  ERROR (app:1890) - Exception on /api/movies [GET]`
```
**Exception on /api/movies [GET]**
```
Traceback (most recent call last):

  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1949, in full_dispatch_request

    rv = self.dispatch_request()

  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1935, in dispatch_request

    return self.view_functions[rule.endpoint](**req.view_args)

  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 467, in wrapper

    resp = resource(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 89, in view

    return self.dispatch_request(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 582, in dispatch_request

    resp = meth(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/api/utils.py", line 30, in wrapper

    return actual_method(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/api/movies/movies.py", line 32, in get

    postprocessMovie(item)

  File "/app/bazarr/bin/bazarr/api/utils.py", line 202, in postprocessMovie

    item[\'subtitles\'] = sorted(item[\'subtitles\'], key=itemgetter(\'name\', \'forced\'))

TypeError: \'<\' not supported between instances of \'NoneType\' and \'str\
```

_Originally posted by @vysmaty in https://github.com/morpheus65535/bazarr/issues/1679#issuecomment-1013744493_

### **Complementary debugg log**
[bazarr.log](https://github.com/morpheus65535/bazarr/files/7876920/bazarr.log)

### Apology
I'm sorry, it originally seemed to me that this was the same issue.

**Comment by morpheus65535** [comment]
Do you have Brazilian, Spanish or Chinese subtitles?

**Comment by vysmaty** [comment]
> Do you have Brazilian, Spanish or Chinese subtitles?

I do not think so. Mostly Czech or English.

I tried to recurse through folders for external. There seems to be no Brazilian, Spanish or Chinese.
`Get-ChildItem -Exclude *.nfo,*.txt,*.jpg,*.png -recurse |?{$_.PSIsContainer -eq $false -and $_.length -lt 500000}`

But I don't know if they're inside the movie file. For example, there will probably be at least Japanese ones. Do you thought inside too?

**Comment by morpheus65535** [comment]
Not sure, I'm kind of driving blind here... could you send me your bazarr.db in private on Discord?

**Comment by vysmaty** [comment]
> Not sure, I'm kind of driving blind here... could you send me your bazarr.db in private on Discord?

Sure. I did so. Thank you.

**Comment by morpheus65535** [comment]
I wrote back some questions to you on Discord to speed up the debug process.

**Comment by vysmaty** [comment]
The error was caused by an invalid language code in the external subtitle name. Namely: `*.old.srt`.

After renaming> `*.baksrt` or valid "*.eng.srt`
`Run Update all Movie Subtitles from disk`

All movies are now listed properly.

**Comment by morpheus65535** [comment]
Should be fixed in upcoming 1.0.3-beta.12. Unsupported languages won't be indexed anymore.
